src/3p-frame.js

```diff
@@ -135,6 +135,9 @@ export function getIframe(
     // request completes.
     iframe.setAttribute('allow', 'sync-xhr \'none\';');
   }
+  if (isExperimentOn(parentWindow, 'sandbox-ads')) {
```

Comment:
Can we add an option to getIframe allowing for control of this behavior (and ideally sync-xhr as well) so that we can run an A/B experiment via the AdSense/Doubleclick FF impls?  This will allow us to guarantee expected results and make canarying the next release simpler.  I'm happy to add this as a subsequent PR if you prefer...

Reply:
Agree. If you could do this in a follow up it would be great!